recipes/r-network/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set version = '1.13.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-network
+  version: {{ version|replace("-", "_") }}
+
+source:
+  fn: network_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/network_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/network/network_{{ version }}.tar.gz
+
+  sha256: 7a04ea89261cdf32ccb52222810699d5fca59a849053e306b5ec9dd5c1184f87
+
+build:
+  number: 0
+  skip: true  # [win32]
+
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - r-base
+    - posix                # [win]
+    - {{native}}toolchain  # [win]
+    - gcc                  # [not win]
+
+  run:
+    - r-base
+    - libgcc  # [not win]
+
+test:
+  commands:
+    - $R -e "library('network')"  # [not win]
+    - "\"%R%\" -e \"library('network')\""  # [win]
+
+about:
+  home: http
+  license: GPL (>= 2)
+  summary: Tools to create and modify network objects.  The network class can represent a range
+    of relational data types, and supports arbitrary vertex/edge/graph attributes.
+  license_family: GPL3
+  license_file: '{{ environ["PREFIX"] }}\/lib\/R\/share\/licenses\/GPL-3'  # [unix]
+  license_file: '{{ environ["PREFIX"] }}\R\share\licenses\GPL-3'  # [win]
+extra:
+   recipe-maintainers:
+    - MathiasHaudgaard
+    - FrodePedersen
+    - ArneKr
+    - Johanneskoester
+    - Bgruening
```

Comment:
@MathiasHaudgaard these should ideally be the real names, the capital letter in the beginning in wrong in most the cases :( Have you used a modified version of the wrapper?